src/components/radios/index.md.njk

```diff
@@ -98,26 +98,25 @@ In services like these, the risk that they will not be noticed is lower because
 
 ### Error messages
 
+Display an error message if none of the radios is selected.
```

Comment:
I'm not 100% on this but isn't this 'if none of the radios _are_ selected'?

Reply:
You're right, but I don't know how to correct it without messing up the commit history :/

Reply:
Nothing wrong with a commit that says 'fix typo' surely 🤪

Reply:
I think you're supposed to rebase rather than make a new commit